platform-ui-skin/src/main/webapp/skin/less/social/skin/social.less

```diff
@@ -35,6 +35,7 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 @import "Search/Style.less";
 @import "popover/popover.less";
 @import "Textarea/Style.less";
+@import "VersionHistoryDrawer/style.less";
```

Comment:
nope, it shouldn't be added to common style imported everywhere. You should import it only in portlets where it's used.

Reply:
**Reminder**: each addon will inherit all files coming from platform-ui and thus can import any file introduced there, see this https://github.com/Meeds-io/notes/blob/develop/notes-webapp/pom.xml#L69-L70

Reply:
OKay